Source/JavaScriptCore/builtins/ArrayConstructor.js

```diff
@@ -202,14 +204,13 @@ function fromAsync(asyncItems  /*, mapFn, thisArg */)
             }
         }
 
-        var result = this !== @Array && @isConstructor(this) ? new this() : [];
-
-        if (!@isUndefinedOrNull(usingAsyncIterator))
-            return @defaultAsyncFromAsyncIterator(result, usingAsyncIterator.@call(asyncItems), mapFn, thisArg);
+        if (!@isUndefinedOrNull(usingAsyncIterator)) {
+            return @defaultAsyncFromAsyncIterator.@call(this, usingAsyncIterator.@call(asyncItems), mapFn, thisArg);
+        }
```

Comment:
Remove `{` and `}`. WebKit coding style does not use it for one statement.

Reply:
Thank you for your review! Fixed